code/modules/games/saintsandsins_cards.dm

```diff
@@ -0,0 +1,673 @@
+	name = "Crippling Despair"
+	ability = "Target may only cast one Key spell on their next turn."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 8
+	)
+
+/decl/sas_card/spell/spark_of_pain
+	name = "Spark of Pain"
+	ability = "Deals 1 Fire damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 10
+	)
+
+/decl/sas_card/spell/devils_tears
+	name = "Devil's Tears"
+	ability = "Deals 1 Water damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 10
+	)
+
+/decl/sas_card/spell/fist_of_stone
+	name = "Fist of Stone"
+	ability = "Deals 1 Earth damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 10
+	)
+
+/decl/sas_card/spell/cutting_blow
+	name = "Cutting Blow"
+	ability = "Deals 1 Wind damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 10
+	)
+
+/decl/sas_card/spell/touch_of_death
+	name = "Touch of Death"
+	ability = "Deals 1 Decay damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 10
+	)
+
+/decl/sas_card/spell/righteous_accusation
+	name = "Righteous Accusation"
+	ability = "Deals 1 Holy damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 10
+	)
+
+/decl/sas_card/spell/cold_stare
+	name = "Cold Stare"
+	ability = "Nullifies any Fire damage from a spell."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_BASICWITCH = 8
+	)
+
+///Wicked Alchemist Kit
+/decl/sas_card/spell/scorn_of_the_alchemist
+	name = "Scorn of the Alchemist"
+	ability = "Caster converts the damage type of another spell to one of their choosing."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/a_glass_darkly
+	name = "A Glass Darkly"
+	ability = "Doubles the damage of the caster's next KEY spell."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/panacea
+	name = "Panacea"
+	ability = "Restores 2 health to caster and one other Heretic."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 8
+	)
+
+/decl/sas_card/spell/scrawling_spellbook
+	name = "Scrawling Spellbook"
+	ability = "All Heretics draw one card."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/grotesque_experiment
+	name = "Grotesque Experiment"
+	ability = "Nullifies another Heretic's spell and they draw a card."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 5
+	)
+
+/decl/sas_card/spell/skin_rot
+	name = "Skin Rot"
+	ability = "Deals 3 Decay damage to all Heretics."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/rosemary_and_thyme
+	name = "Rosemary and Thyme"
+	ability = "Purges any damage-over-time spells on caster."
```

Comment:
```suggestion
	ability = "Nullifies all damage-over-time spells targeting the caster."
```
Might need to clarify this for spells that target all heretics.

Reply:
This is specifically for active DoTs, will adjust to be clearer though.